.circleci/config.yml

```diff
@@ -1630,6 +1635,23 @@ jobs:
       - store_test_results:
           path: test/test-results/junit.xml
 
+  test-integration:
+    executor: node-browsers-medium-plus
+    steps:
+      - run: *shallow-git-clone
+      - run: sudo corepack enable
+      - attach_workspace:
+          at: .
+      - run:
+          name: test:integration:coverage
+          command: yarn test:integration:coverage
+      - persist_to_workspace:
+          root: .
+          paths:
+            - coverage
+      - store_test_results:
+          path: test/test-results/integration/junit.xml
+
```

Comment:
:information_source:  as info (no change required), in the future it's planned to move the unit test ci jobs to github actions. I think we'll want to do the same with integrations tests?

https://github.com/MetaMask/MetaMask-planning/issues/2449